qiskit/transpiler/passes/synthesis/high_level_synthesis.py

```diff
@@ -887,6 +892,107 @@ def run(self, high_level_object, coupling_map=None, target=None, qubits=None, **
         return decomposition
 
 
+class QFTSynthesisFull(HighLevelSynthesisPlugin):
```

Comment:
do we see these classes in the docs?

Reply:
Thanks for catching this! I have extended the section on available plugins in [4ae4c9e](https://github.com/Qiskit/qiskit/pull/11463/commits/4ae4c9eef02e2e566a1cac87c9b55c068881cf68)